tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/Emitter.scala

```diff
@@ -444,6 +460,11 @@ final class Emitter private (semantics: Semantics, outputMode: OutputMode,
 
   // Helpers
 
+  private def mergeVersions(v1: Option[String],
+      v2: Option[String]): Option[String] = {
+    v1.flatMap(s1 => v2.map(s2 => s1 + s2))
```

Comment:
Nope :( Consider:

```scala
mergeVersions("1", "11") // --> 111
mergeVersions("11", "1") // --> 111
```

Bad. Maybe we should allow `Any` as version and just rely on the equality method? Then you could just use a tuple here? (or do we need versions to be serializable?)